.github/workflows/deploy-production.yml

```
@@ -109,6 +121,42 @@ jobs:
+      ECS_SERVICE: ${{ vars.ECS_SERVICE_MLLP_SERVER_PRODUCTION }}
+      AWS_REGION: ${{ vars.API_REGION_PRODUCTION }}
+    secrets:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      SENTRY_AUTH_TOKEN: ${{ secrets.SENTRY_AUTH_TOKEN }}
+      SENTRY_ORG: ${{ secrets.SENTRY_ORG }}
+      DOCKERHUB_USERNAME: ${{ secrets.DOCKERHUB_USERNAME }}
+      DOCKERHUB_TOKEN: ${{ secrets.DOCKERHUB_TOKEN }}
+
+  infra-hl7-notification-routing-stack-prod:
+    if: needs.files-changed.outputs.infra-hl7-notification-routing-stack == 'true'
```

actionlint (1.7.4)

property "infra-hl7-notification-routing-stack" is not defined in object type {api: string; fhir-converter: string; infra-lambdas: string; terminology: string}

(expression)